libraries/standard/mqtt/src/mqtt_lightweight.c

```diff
@@ -343,37 +346,6 @@ static uint8_t * encodeString( uint8_t * pDestination,
 
 /*-----------------------------------------------------------*/
 
-static int32_t recvExact( MQTTTransportRecvFunc_t recvFunc,
```

Comment:
Is this function not needed?

Reply:
No. It is not used anywhere and I see that PR #901 brings a read function